packages/loader/container-definitions/src/loader.ts

```diff
@@ -285,13 +285,55 @@ export enum LoaderHeader {
     version = "version",
 }
 
+export interface IContainerLoadMode {
+    opsBeforeReturn?:
+        /*
+         * No trailing ops are applied before container is returned.
+         * Default value.
+         */
+        | undefined
+        /*
+         * Only cached trailing ops are applied before returning container.
```

Comment:
Would this mean the persisted op cache only or also ops in the trees/latest request in the ODSP case when we fetch a snapshot?

Reply:
All cached ops. Loader is not aware here, so it's up to driver how to interpret that. ODSP returns all cached ops it knows about, but some other driver may have different behavior. Note that op caching (through host) or snapshot paradigms are not known at loader level.

Reply:
Great, that makes sense.